components/tyk-enterprise-portal/values.yaml

```diff
@@ -0,0 +1,117 @@
+# Default values for tyk-enterprise-portal.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Enterprise Portal will only bootstrap if the tyk-enterprise-portal-conf secret
+# is present. The secret is created during the dashboard bootstrapping and
+# be turned on and off using the dash.enterprisePortalSecret bool.
+
+
+# The hostname to bind the Enterprise Portal to.
+hostName: tyk-enterprise-portal.local
+# Enterprise Portal license.
+license: ""
+# Enterprise Portal admin information.
```

Comment:
i guess this comment does not belong here?

Reply:
Fixed